conans/test/integration/command/new_test.py

```diff
@@ -45,6 +45,21 @@ def test_new_missing_definitions(self):
         client.run("new cmake_lib -d version=myversion", assert_error=True)
         assert "Missing definitions for the template. Required definitions are: 'name', 'version'" in client.out
 
+    def test_new_basic_template(self):
+        tc = TestClient()
+        try:
+            tc.run("new basic")
+            tc.assert_in_file_contents("conanfile.py", ['# self.requires("zlib/1.2.13")'])
+        except Exception as e:
+            assert False, f"conan new basic should work without any extra arguments: {e}"
+
+        tc.run("new basic -d name=mygame -d requires=math/1.0 -d requires=ai/1.0 -f")
+        tc.assert_in_file_contents("conanfile.py", [
+            'self.requires("math/1.0")',
+            'self.requires("ai/1.0")',
+            'name = "mygame"'
+        ])
```

Comment:
```suggestion
        conanfile = tc.load("conanfile.py")
        assert 'self.requires("math/1.0")' in conanfile
        assert 'self.requires("ai/1.0")' in conanfile
        assert 'name = "mygame"' in conanfile
```

There is an evil side in moving asserts to helpers (even though I also do it often), and is that those asserts do not report the same, and it is worse experience for developers when they fail.
It doesn't seem the helper really saves many lines, and the result is equally readable?

Reply:
Oh I missed the fact that TestClient.load did exactly what I wanted to! Thanks for the headsup :)